components/test_raftstore/src/node.rs

```diff
@@ -269,6 +269,7 @@ impl Simulator for NodeCluster {
                 .max_total_size(cfg.server.snap_max_total_size.0)
                 .encryption_key_manager(key_manager)
                 .max_per_file_size(cfg.raft_store.max_snapshot_file_raw_size.0)
+                .enable_multi_snapshot_files(true)
```

Comment:
Why the change?

Reply:
Because I find test cases like `test_server_huge_snapshot_multi_files` may not check expected behavior if `enable_multi_snapshot_files` is set to false. 
These tests are to test if snapshot works fine when we enable multi file snapshot. However, we may not generate multi file snapshot if this switch is closed, so these tests will actually test single file snapshot.